services/compute/virtualmachine/wssd.go

```diff
@@ -199,6 +199,39 @@ func (c *client) Validate(ctx context.Context, group, name string) error {
 	return nil
 }
 
+func (c *client) Precheck(ctx context.Context, group string, vms []*compute.VirtualMachine) (bool, error) {
+	request, err := c.getVirtualMachinePrecheckRequest(group, vms)
+	if err != nil {
+		return false, err
+	}
+	response, err := c.VirtualMachineAgentClient.Precheck(ctx, request)
+	if err != nil {
+		return false, err
+	} else {
```

Comment:
minor: if error return.

code can continue without else. Would keep it cleaner without indentation

Reply:
OK.